File: App2/src/app/_core/_model/pig-disease/pig-disease.ts
export interface PigDisease {
  id: number;
  type: string;
  pigType: string;
  recordDate: string | null;
  recordReason: string;
  recordGuid: string;
  approveDate: string | null;
  approveReason: string;
  approveGuid: string;
  rejectDate: string | null;
  rejectReason: string;
  rejectGuid: string;
  comment: string;
  createDate: string | null;
  createBy: number | null;
  updateDate: string | null;
  updateBy: number | null;
  status: number | null;
  guid: string;
   typeName: string;
  pigTypeName: string;
}

File: PigFarm/Models/ModelsPhase4/PigDisease.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PigFarm.Models
{
    public class PigDisease
    {
        public decimal Id { get; set; }
        public string Type { get; set; }
        public string PigType { get; set; }
        public DateTime? RecordDate { get; set; }
        public string RecordReason { get; set; }
        public string RecordGuid { get; set; }
        public DateTime? ApproveDate { get; set; }
        public string ApproveReason { get; set; }
        public string ApproveGuid { get; set; }
        public DateTime? RejectDate { get; set; }
        public string RejectReason { get; set; }
        public string RejectGuid { get; set; }
        public string Comment { get; set; }
        public DateTime? CreateDate { get; set; }
        public decimal? CreateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public decimal? UpdateBy { get; set; }
        public decimal? Status { get; set; }
        public string Guid { get; set; }
    }
}
